Assets/Scripts/PlayerShooting.cs

```diff
@@ -33,13 +33,7 @@ void FireProjectile()
             return;
         }
 
-        var ammoBody = pickupAmmunition.ammunition.GetComponent<Rigidbody>();
-        if (!ammoBody)
-        {
-            Debug.LogWarning("Projectile does not have a Rigidbody.", this);
-            return;
-        }
```

Comment:
I think it's wise to keep warnings like these, or if it's more crucial then promoting it to a LogError instead